ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-LEAF2B.cfg

```diff
@@ -548,7 +550,6 @@ ip route vrf Tenant_D_OP_Zone 0.0.0.0/0 10.3.11.4
 ip route vrf Tenant_D_OP_Zone 10.3.11.0/24 Vlan411 name VARP
 !
 ipv6 route vrf Tenant_D_OP_Zone ::/0 2001:db8:311::4 name IPv6-test-2
-ipv6 route vrf Tenant_D_OP_Zone 2001:db8:311::/64 Vlan411 name VARPv6
```

Comment:
Going with my other comment below - what was the initial purpose behind this injected route "when having a prefix for the ipv6 virtual router address"?

Reply:
it was just copy/paste from ipv4, where we _do_ support mask, and the VARP guide tells that when you have a mask on VARP, you need this static route.

Reply:
ack - let's resolve this then!